src/index.html

```diff
@@ -11,12 +12,102 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__top">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+        />
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```